src/components/Navbar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <div>
+      <header>
+        <nav>
+          <Link className="Logo" to="/Home">
+            Math Magicians
+          </Link>
+          <ul className="navList">
+            <li className="navItem">
+              <Link className="navAnchor" to="/Home">
+                Home
```

Comment:
- **[OPTIONAL]** The navbar link should be root path also to match the route in the app component.

  ```
  <Link className="Logo" to="/">
    Math Magicians
  </Link>
  <ul className="navList">
    <li className="navItem">
      <Link className="navAnchor" to="/">
        Home
  ```

Reply:
Oh yes! Thanks for the suggestion, I'll definitely add it 😀